sat/cli/jobstat/main.py

```diff
@@ -25,8 +25,9 @@
 """
 import logging
 import textwrap
+import re
 
-from sat.apiclient import APIError
+from csm_api_client.service.gateway import APIError
 from sat.apiclient.jobstat import JobstatClient
```

Comment:
Micro nit: would prefer a blank line between these two imports. We tend to group our imports as follows:
```
python std lib imports
<empty line>
third-party library imports
<empty line>
imports from the sat package
```

In this case `csm_api_client` is closest to being a third-party package (while it is authored by HPE, it's external to `sat`).

```suggestion
from csm_api_client.service.gateway import APIError

from sat.apiclient.jobstat import 
```